kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/impl/PrometheusDataSetDefBuilderImpl.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dashbuilder.dataset.impl;
+
+import org.dashbuilder.dataset.def.DataSetDef;
+import org.dashbuilder.dataset.def.PrometheusDataSetDef;
+import org.dashbuilder.dataset.def.PrometheusDataSetDefBuilder;
+
+public class PrometheusDataSetDefBuilderImpl extends AbstractDataSetDefBuilder<PrometheusDataSetDefBuilderImpl> implements PrometheusDataSetDefBuilder<PrometheusDataSetDefBuilderImpl> {
+
+    @Override
+    public PrometheusDataSetDefBuilderImpl serverUrl(String serverUrl) {
+        ((PrometheusDataSetDef) def).setServerUrl(serverUrl);
```

Comment:
```suggestion
        ((PrometheusDataSetDef) super.def).setServerUrl(serverUrl);
```
Consider adding a `super` for this and the other method below.
IMO it improves readability.